spec/models/miq_report/formats_spec.rb

```diff
@@ -43,5 +43,12 @@
         end
       end
     end
+
```

Comment:
Since this spec is pretty small, perhaps it would be good to add another case that tests when there isn't the specific override. Just to be safe.